custom_components/places/helpers.py

```
@@ -95,11 +127,26 @@ def write_sensor_to_json(
+    """Remove the Places ``(since HH:MM)`` or ``(since MM/DD)`` suffix.
+
+    Args:
+        orig_state: Sensor state that may include a trailing ``since`` suffix.
+
+    Returns:
+        State string without the generated suffix.
+    """
     return re.sub(r" \(since \d\d[:/]\d\d\)", "", orig_state)
 
 
 def safe_truncate(val: Any, max_len: int) -> str:
```

Ruff (0.15.12)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `val`

(ANN401)